### PR TITLE
do not log group already exists error message

### PR DESCRIPTION
**Pull request by Marianasilvago**
### What

Cleanup logging group already exists error in logs on startup.

### Who can review

Anyone.

### CODE REVIEW

File: models/errors.go [hunk 1]
```diff
@@ -41,6 +42,15 @@ func NewValidationError(ctx context.Context, code string, description string) *E
...
 	return err
 }
+//IsGroupExistsError validates if the err occurred because group already exists
+func IsGroupExistsError(err error) bool {
+	var cognitoErr awserr.Error
+	if errors.As(err, &cognitoErr) && cognitoErr.Code() == cognitoidentityprovider.ErrCodeGroupExistsException {
+		return true
+	} else {
+		return false
+	}
```
Comment (pidah) [inline models/errors.go]: just a nit, can we change it to:
```
if errors.As(err, &cognitoErr) && cognitoErr.Code() == 
cognitoidentityprovider.ErrCodeGroupExistsException {
		return true
}
return false
```

**Review by pidah — COMMENTED** [review]
LGTM

**Review by Marianasilvago — COMMENTED** [review]
LGTM

**Review by pidah — DISMISSED** [review]

**Review by mike-kelly-methods — APPROVED** [review]
👍

**Pull request merged**